<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6JOMMLI" failures="1" tests="4" name="Test" time="7.782" errors="0" timestamp="2025-08-02T11:46:49 IST">
  <testcase classname="api.test.userTest2" name="testPostUser" time="3.295"/>
  <testcase classname="api.test.userTest2" name="getUser" time="1.132"/>
  <testcase classname="api.test.userTest2" name="updatedUser" time="1.123"/>
  <testcase classname="api.test.userTest2" name="deleteUser" time="1.131">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.userTest2.deleteUser(userTest2.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteUser -->
</testsuite> <!-- Test -->
